bin/cqlsh.py

```diff
@@ -2090,13 +2103,19 @@ def do_login(self, parsed):
 
         auth_provider = PlainTextAuthProvider(username=username, password=password)
 
-        conn = Cluster(contact_points=(self.hostname,), port=self.port, cql_version=self.conn.cql_version,
+        kwargs = {}
+        if self.cloudconf is None:
+                kwargs['contact_points'] = (self.hostname,)
+                kwargs['port'] = self.port
+                kwargs['ssl_options'] = self.conn.ssl_options
+                kwargs['load_balancing_policy'] = WhiteListRoundRobinPolicy([self.hostname])
+        conn = Cluster(cql_version=self.conn.cql_version,
                        protocol_version=self.conn.protocol_version,
                        auth_provider=auth_provider,
-                       ssl_options=self.conn.ssl_options,
-                       load_balancing_policy=WhiteListRoundRobinPolicy([self.hostname]),
```

Comment:
that's a good question, i'm guessing login function is less used (since you can put the user/name in the command line)
so no one on upstream fixed it.

Reply:
i.e. it can be aligned, but it's unrelated to this specific change